newIDE/app/src/UI/Layout.js

```diff
@@ -171,6 +171,7 @@ export const ResponsiveLineStackLayout = ({
     <ColumnStackLayout
       id={id}
       noMargin={noMargin || noColumnMargin}
+      justifyContent={justifyContent}
```

Comment:
we were never passing down the justifyContent to the column, when the mobile size kicks in.
I'm surprised we never needed this. I needed it to design properly a text being centered when going mobile

Reply:
Feels a bit risky to me. I think there was a workaround using a Line as an intermediary component. But if you see no weird change, let's go for it.
Also, should it be a justifyContentWhenColumn?

Reply:
It does feel risky indeed, I'll try a workaround

Reply:
eventually removed it as didn't remember why I needed it, it seems like I got it sorted without